src/go/rpk/pkg/cli/cluster/txn/describe_producers.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2022 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package txn
+
+import (
+	"time"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/kafka"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/twmb/franz-go/pkg/kadm"
+)
+
+func newDescribeProducersCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var (
+		topics     []string
+		partitions []int32
+	)
+	cmd := &cobra.Command{
+		Use:   "describe-producers",
```

Comment:
Note: this should have the --all flag rather than defaulting to all toppars in the cluster